lib/utils/testLauncherHelper.js

```diff
@@ -248,17 +191,14 @@ async function launchChild(cmdArgv, ownArgv, options, runMode) {
 	const [command, ...args] = cmdArgv;
 
 	if (debugPort) {
-		const debugArgs = `--inspect${inspectBrk ? '-brk' : ''}=${debugPort}`;
-
-		return cp.fork(command, args, {
-			...options,
-			execArgv: [
-				debugArgs,
-			],
-			cwd: normalize(process.cwd()),
-			execPath: normalize(process.execPath),
-			silent: true,
-		});
+		args.unshift('--inspect-brk');
+		args.unshift(`--inspect${inspectBrk ? '-brk' : ''}=${debugPort}`);
+	}
+
+	if (runMode === INTERACTIVE) {
+		args.unshift('--experimental-repl-await');
+		options.stdio = ['pipe', 'pipe', 'pipe'];
```

Comment:
from nodejs docs:
'pipe' - equivalent to ['pipe', 'pipe', 'pipe'] (the default)

Reply:
If only you knew how many times I have changed this line during my experiments with different io methods :)
... i'll change that.